### ISSUE TITLE
Add readme.md file 

### ISSUE THREAD

**Issue by akashkroy** [Noman5237/gitea-sonarqube-community-bot#1]
Can you add instructions on how to run this ?

**Comment by Noman5237** [comment]
yeah sure I will try to do it asap, I just can't find enough time to do it

Basically you run the docker compose file which starts the gitea with postgres server for testing (If you need it, do it if you don't have any gitea server already)

Set the environment variables defined in the [config](https://github.com/Noman5237/gitea-sonarqube-community-bot/blob/develop/src/globals.js)

Then run the docker compose for the bot and the bot will start, currently the bot only have strategies to run java projects for maven and gradle, but you can add your own strategies in this [file](https://github.com/Noman5237/gitea-sonarqube-community-bot/blob/develop/src/sonarqube/sonarqube.strategy.js)

**Comment by lichuanqi** [comment]
same issue

**Comment by Noman5237** [comment]
Fine i will finally complete this project and update the docs lmao
